### PR TITLE
Feature/3317 drawer body api docs

### PR DESCRIPTION
**Pull request by jeffvg**
Fixes # BLUI-3317

#### Changes proposed in this Pull Request:
- Add Drawer Body api docs
- 
- 

#### Screenshots:
image

#### To Test:
- yarn start:reactdev
- navigate to Drawer->Drawer Body >API DOCS (tab)

- Note the shared props in table is not hooked up yet and needs built out to fix this PR and PR Added Drawer Nav Group api docs #469

### CODE REVIEW

File: docs/src/components/componentList/drawerBodyDocsComponent/DrawerBodyDocsComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (bkarambe) [inline docs/src/components/componentList/drawerBodyDocsComponent/DrawerBodyDocsComponent.tsx:1]: Once you the base branch to resolve merge conflicts, this component file won't be needed and you can directly use DrawerBodyMarkdown in the navigation.

- [ ] Use DrawerBodyMarkdown directly in navigation

**Review by bkarambe — CHANGES_REQUESTED** [review]
- [x] Please resolve merge conflicts

File: docs/src/components/componentList/drawerBodyDocsComponent/DrawerBodyDocsComponent.tsx (hunk 1 above)
Comment (jeffvg) [reply docs/src/components/componentList/drawerBodyDocsComponent/DrawerBodyDocsComponent.tsx:1]: Not sure what you mean. You have updated the react-dev-docs? The merge conflict is here...

image

**Comment by jeffvg** [comment]
I see the changes from you today, quite a bit considering we were using components. When was it decided to change the implementation of these?

image

**Review by bkarambe — APPROVED** [review]
Looks good.

**Pull request merged**